server/routes/users.js

```diff
@@ -4,6 +4,6 @@ const router = express.Router()
 const {getAllUsers,addUser,getUser , updateGeneralDetails,publishVersion,deleteUser} = require('../controllers/users')
 
 router.route('/').get(getAllUsers).post(addUser);
-router.route('/:userName').get(getUser).post(publishVersion).put(updateGeneralDetails).delete(deleteUser);
+router.route('/:userName').get(getUser).post(publishVersion).patch(updateGeneralDetails).delete(deleteUser);
```

Comment:
Why are we using patch?

Reply:
bhaiya I read that it is better to use patch than put to update resources